plugins/chat/assets/javascripts/discourse/components/chat-channel.js

```diff
@@ -201,6 +198,18 @@ export default class ChatLivePane extends Component {
         }
 
         this.scrollToBottom();
+
+        if (result.threads) {
```

Comment:
Maybe I shouldn't do this and the `thread_tracking_overview` below until the next PR since I don't use it yet in the UI, thoughts?

Reply:
I think that's ok, will help us catch issues anyways